src/test/java/POM/LoginPage.java

```diff
@@ -0,0 +1,54 @@
+package POM;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class LoginPage extends  basePage{
```

Comment:
Class basePage is missing

![image](https://user-images.githubusercontent.com/104761482/200170176-f0e5f800-4860-44e7-89e0-bed110f85ef0.png)

Create class or delete basePage

![image](https://user-images.githubusercontent.com/104761482/200170201-4083b2a4-4cc8-40c3-b546-f188ae225361.png)